src/features/articles.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit";
```

Comment:
to be more concise, you can consider naming slice files accordingly, like "articleSlice.js"